app/assets/json/translations/locale-en.json

```diff
@@ -819,6 +826,39 @@
             "zipcode": "Zipcode"
         },
         "listings": {
+            "habitat": {
+                "payments": {
+                    "title": "Payments",
+                    "p1": "Your monthly housing payment will be 30% of your gross monthly income. The housing payment includes your mortgage, property tax, insurance, HOA fee, and utilities."
+                },
+                "sweat_equity": {
+                    "title": "Sweat Equity",
+                    "p1": "You do not need to pay a down payment. You do need to complete 500 hours of sweat equity. Sweat equity is volunteer hours that you and your friends and family do.",
+                    "p2": "If you are concerned about your ability to complete 500 hours of sweat equity, reach out to Habitat to ask how they can help."
+                },
+                "application_process": {
+                    "title": "Application Process",
+                    "p1": "These units are offered by <a target='_blank' href='{{habitatLink}}'>Habitat for Humanity Greater San Francisco</a> in partnership with MOHCD",
+                    "ol1": "You must go to a <a target='_blank' href='{{infoSessionLink}}'>Habitat for Humanity information session</a>.",
+                    "ol2": "You will submit your application on DAHLIA.",
+                    "ol3": "MOHCD will run the lottery and post results here on DAHLIA.",
+                    "ol4": "In lottery rank order, Habitat for Humanity Greater San Francisco will do a credit and background check.",
+                    "ol5": "MOHCD will check to see if you qualify for the listing. If you qualify, MOHCD will issue you a conditional approval.",
+                    "ol6": "You must complete 500 hours of sweat equity.",
+                    "ol7": "Habitat Greater San Francisco will work with you to approve your first mortgage loan.",
+                    "ol8": "MOHCD will work with you to issue a final approval.",
+                    "ol9": "MOHCD and Habitat Greater San Francisco will deliver closing documents to escrow.",
+                    "ol10": "You can move into your new home!"
+                },
+                "income_range": {
+                    "title": "Income Range",
+                    "p1": "To be eligible for a Habitat for Humanity Greater San Franscisco home in this listing, your household income needs to be between 69% and 109% AMI as listed below.",
```

Comment:
Confirmed that the AMI % is not connected to salesforce, so this is content that we'll have to update manually, like others.